File: RLE.API/Models/Municipality.cs
using System;
using System.Collections.Generic;

namespace RLE.API.Models
{
    public partial class Municipality
    {
        public Municipality()
        {
            // Employee = new HashSet<User>();
            // EnrolmentCenter = new HashSet<EnrolmentCenter>();
            // TrainingClass = new HashSet<TrainingClass>();
        }

        public int Id { get; set; }
        public int? CityId { get; set; }
        public string Name { get; set; }
        public string Code { get; set; }
        public int PollingPlaces { get; set; }

        public  City City { get; set; }
        public virtual  ICollection<User> Employee { get; set; }
        public  virtual  ICollection<EnrolmentCenter> EnrolmentCenters { get; set; }
        public virtual  ICollection<TrainingClass> TrainingClass { get; set; }
    }
}

File: RLE-SPA/src/app/_models/municipality.ts

export class Municipality {
  public id: number;

  public cityId: number;

  public name: string;

  public code: string;

  public pollingPlaces: number;

  constructor(
    id?: number,
    cityId?: number,
    name?: string,
    code?: string,
    pollingPlaces?: number,
    ) {
    this.id = id || 0;
    this.cityId = cityId || null;
    this.name = name || '';
    this.code = code || '';
    this.pollingPlaces = pollingPlaces || 0;
  }
}
